src/+otp/+sanzserna/SanzSernaProblem.m

```diff
@@ -0,0 +1,27 @@
+classdef SanzSernaProblem < otp.Problem
+    methods
+        function obj = SanzSernaProblem(timeSpan, y0, parameters)
+            obj@otp.Problem('Sanz-Serna', [], timeSpan, y0, parameters);
+        end
+    end
+    
+    methods (Access=protected)
+        
+        function onSettingsChanged(obj)
+            
+            D = spdiags(ones(obj.NumVars, 1) * [obj.NumVars, -obj.NumVars], [-1, 0], obj.NumVars, obj.NumVars);
```

Comment:
Here this looks like, but is not (from my understanding) a downwind finite difference, right? The letter "D" implies that it is, however. We might not want to create that impression.

Reply:
This is a first order upwind finite difference matrix.  D for differentiation matrix I guess.

Reply:
If it is, then should it not be divided by h? Otherwise what is here is wrong